### PR TITLE
i18n_static_pages

### PR DESCRIPTION
**Pull request by oakprofessor**
screenshot from 2018-07-30 23-20-42

### CODE REVIEW

**Comment by TaiTran36** [comment]
ok

**Comment by oakprofessor** [comment]
ready

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<% provide(:title, "ShaRoom") %>
+<% provide :title, "ShaRoom" %>
 <div id="container-full">
   <!-- noi dung -->
 	<div class="content" style="height: 985px">
```
Comment (hwanngo) [inline app/views/static_pages/home.html.erb]: Indent

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
   def full_title page_title = ""
-    base_title = "Chia sẻ thông tin - Tìm kiếm phòng trọ"
+    base_title = t(".app_title")
```
Comment (hwanngo) [inline app/helpers/application_helper.rb]: `(` `)`

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -1,35 +1,35 @@
 <div class="footer">
   <div class="contact">
     <div class="forMe">
-      <h3><strong>về chúng tôi</strong></h3>
+      <h3><strong><%= t(".foot_about") %></strong></h3>
```
Comment (hwanngo) [inline app/views/layouts/_footer.html.erb]: `(` `)`

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -13,9 +13,18 @@
         <nav>
           <ul class="nav navbar-nav navbar-right">
             <li><span class="glyphicon glyphicon-user"></span></li>
-            <li><a href="#">Sign up</a></li>
+            <li><a href="#"><%= t(".sign_up") %></a></li>
```
Comment (hwanngo) [inline app/views/layouts/_header.html.erb]: `(` `)`

**Comment by oakprofessor** [comment]
ready

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -1,181 +1,163 @@
-<% provide(:title, "ShaRoom") %>
+<% provide :title, "ShaRoom" %>
 <div id="container-full">
-  <!-- noi dung -->
 	<div class="content" style="height: 985px">
-		<div class="container">    
+		<div class="container">
 			<div class="row">
```
Comment (ghost) [inline app/views/static_pages/home.html.erb]: file bạn chuyển indent từ tab sang 2spaces nhé

**Comment by ghost** [comment]
commented

**Comment by oakprofessor** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**